functions/utils.py

```diff
@@ -1,13 +1,31 @@
+﻿#------------------------------------------------------------------------------
+# Copyright 2016 Esri
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#------------------------------------------------------------------------------
+
 __all__ = ['isProductVersionOK',
            'computePixelBlockExtents',
            'computeCellSize',
            'Projection',
            'Trace',
-           'ZonalAttributesTable',]
+           'ZonalAttributesTable',
+           'projectCellSize',]
 
 
 # ----- ## ----- ## ----- ## ----- ## ----- ## ----- ## ----- ## ----- #
 
+degreeToMeter = 111319.49079327357264771338267056

```

Comment:
Whenever I see this type of value hard-wired to Earth I get nervous.  This should be calculated from the semi-major radius such that it can work on other non-Earth bodies. If you are truly support the spatial reference here, hopefully this conversion value can be read from the SR.